backend/package.json

```diff
@@ -44,7 +44,7 @@
     "connect-redis": "4.0.4",
     "convict": "6.2.4",
     "cors": "2.8.5",
-    "dd-trace": "2.13.0",
```

Comment:
`import-in-the-middle` is not directly upgraded?

Also, how do we test if the upgrade works? Pass CI == OK?

Reply:
Not directly upgraded because it only exists in `package-lock.json` as a dep under `dd-trace`.

I'm taking a leap of faith here because the upgrade was a minor version upgrade and it was done for the other folders but not this.